.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,408 @@
+        if: needs.build-api.result == 'success'
+        uses: actions/download-artifact@v4
+        with:
+          name: api-build
+          path: apps/api/
+
+      - name: Deploy to staging
+        id: deploy
+        env:
+          STAGING_DEPLOY_TOKEN: ${{ secrets.STAGING_DEPLOY_TOKEN }}
+          STAGING_ENDPOINT: ${{ vars.STAGING_ENDPOINT || 'https://deploy-api.staging.nester.dev' }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:11:42: Double quote to prevent globbing and word splitting

(shellcheck)